### ISSUE TITLE
IPython not working after wandb import

### ISSUE THREAD

**Issue by kantholtz** [wandb/client#1253]
* Weights and Biases version: 0.10.2
* Python version: 3.8.5
* Operating System: Linux

### Description

After importing wandb, IPython can no longer be embedded. Without `import wandb` everything works fine.

### What I Did
```
Python 3.8.5 (default, Aug  5 2020, 08:36:46)
[GCC 7.3.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import wandb
>>> __import__('IPython').embed()                                                                                                                                                                                                                                 
Traceback (most recent call last):
File "<stdin>", line 1, in <module>
File ".../conda/envs/ryn/lib/python3.8/site-packages/IPython/__init__.py", line 56, in <module>
from .terminal.embed import embed
File ".../conda/envs/ryn/lib/python3.8/site-packages/IPython/terminal/embed.py", line 16, in <module>
from IPython.terminal.interactiveshell import TerminalInteractiveShell
File ".../conda/envs/ryn/lib/python3.8/site-packages/IPython/terminal/interactiveshell.py", line 21, in <module>
from prompt_toolkit.formatted_text import PygmentsTokens
ModuleNotFoundError: No module named 'prompt_toolkit.formatted_text'
```

**Comment by cvphelps** [comment]
Hi @kantholtz thanks for reporting this. @raubitsj could you please take a look

**Comment by raubitsj** [comment]
@kantholtz , thanks for reporting this issue.

Because of a library we are including we use an old version of prompt_toolkit which is vendored at: `wandb/vendor/prompt_toolkit/`

I am guessing that causing an issue with IPythons preferred prompt_toolkit version.  We will look into a solution to prevent our vendored version from impacting the system library dependancies.

**Comment by kantholtz** [comment]
Alright, thank you very much for your quick replies!

**Comment by willprice** [comment]
Any news on this @raubitsj ? I just wanted to try out wandb but could not due to this issue breaking `ipdb`.

**Comment by felixkreuk** [comment]
Latest version of wandb is not working with ipdb for me as well.

**Comment by kantholtz** [comment]
Bump - because `pdb` also uses this old readline and control commands (such as C-a, C-e, C-p, etc.) don't work.

**Comment by sydholl** [comment]
This issue has been solved in Client Release 0.10.11
